protocol/qrcode/federation_exchange.json

```diff
@@ -31,7 +26,9 @@
             "properties": {
                 "value": {
                     "type": "string",
-                    "description": "The challenge value, which is a string of 32 random bytes encoded in base64 format"
+                    "contentEncoding": "hex",
+                    "pattern": "^[0-9a-fA-F]{64}$",
+                    "$comment": "A 32 bytes array encoded in hexadecimal"
```

Comment:
Not that you need to change anything here, just curious: Why did you chose to go with hex instead of base64?

Reply:
Honestly, when we discussed the data authentication protocol we just described to use either base64 or hex and the backend guys started to implement it with hex..

Reply:
huh okay, interesting :)